app/services/VatKnownFactsRetrievalService.scala

```diff
@@ -64,7 +64,7 @@ object VatKnownFactsRetrievalService {
 
   case object DeregisteredUser extends GetVatKnownFactsSuccess
 
-  case class VatKnownFacts(vatRegistrationDate: String, businessPostCode: Option[String], isOverseas: Boolean) extends GetVatKnownFactsSuccess
+  case class VatKnownFacts(effectiveRegistrationDate: String, businessPostCode: Option[String], isOverseas: Boolean) extends GetVatKnownFactsSuccess
```

Comment:
The `/known-facts` endpoint is already in use by another team so if we change the name of this field here it would cause failures for them. Could you reset this back to how it was (and then the corresponding change on line 42)